orchestra/static/orchestra/common/components/modules.js

```diff
@@ -0,0 +1,11 @@
+(function () {
```

Comment:
Your idea sounds good to me.  Let's not move it into `task` until we get the sense that makes sense :).